src/google/protobuf/compiler/java/java_message_builder.cc

```diff
@@ -473,26 +473,32 @@ GenerateCommonBuilderMethods(io::Printer* printer) {
     "public Builder setField(\n"
     "    com.google.protobuf.Descriptors.FieldDescriptor field,\n"
     "    Object value) {\n"
+    "  //@@protoc_insertion_point(field_modifier_scope_after:$full_name$.setField)\n"
```

Comment:
Ok, I'll try to find a way to do this patch without needing insertion points in this file.